cypress/integration/regions_test/regions_export_test.js

```diff
@@ -0,0 +1,93 @@
+describe("Regions Export Test", function(){
```

Comment:
Could you repeat these tests for the first region result, using the buttons inside `.results-footer`? They give different files (if you have more than one region results - let me know if this isn't the case for the biotestmine example) and could fail while the exports above for all results continue to work.

In this case it would be useful to have more descriptive filenames for the fixtures, like `all_regions_result.csv` and `region_result.csv`.